docs/fcrepo.properties

```diff
@@ -0,0 +1,7 @@
+fcrepo.activemq.directory=/data/activemq
+fcrepo.home=/data
```

Comment:
Shouldn't this be `/data/fcrepo-home`? I'm a little confused as to what's being mapped where.

Reply:
In this setup, `/data` in the Fedora Docker container is Fedora home: https://github.com/fcrepo-exts/fcrepo-docker/pull/13/files#diff-dec5aef1eb56327aab4a81fa2f9a69ecbede5b3ea1413e93ccb7f1dd7ea75018R17